doc/classes/OS.xml

```diff
@@ -16,30 +16,32 @@
 			<param index="0" name="text" type="String" />
 			<param index="1" name="title" type="String" default="&quot;Alert!&quot;" />
 			<description>
-				Displays a modal dialog box using the host OS' facilities. Execution is blocked until the dialog is closed.
+				Displays a modal dialog box using the host platform's implementation. The engine execution is blocked until the dialog is closed.
 			</description>
 		</method>
 		<method name="close_midi_inputs">
 			<return type="void" />
 			<description>
-				Shuts down system MIDI driver.
+				Shuts down the system MIDI driver. See also [method open_midi_inputs] and [method get_connected_midi_inputs].
 				[b]Note:[/b] This method is implemented on Linux, macOS and Windows.
 			</description>
 		</method>
 		<method name="crash">
 			<return type="void" />
 			<param index="0" name="message" type="String" />
 			<description>
-				Crashes the engine (or the editor if called within a [code]@tool[/code] script). This should [i]only[/i] be used for testing the system's crash handler, not for any other purpose. For general error reporting, use (in order of preference) [method @GDScript.assert], [method @GlobalScope.push_error] or [method alert]. See also [method kill].
+				Crashes the engine (or the editor if called within a [code]@tool[/code] script).
+				[b]Note:[/b] This method should [i]only[/i] be used for testing the system's crash handler, not for any other purpose. For general error reporting, use (in order of preference) [method @GDScript.assert], [method @GlobalScope.push_error] or [method alert]. See also [method kill].
```

Comment:
```suggestion
				[b]Note:[/b] This method should [i]only[/i] be used for testing the system's crash handler, not for any other purpose. For general error reporting, use (in order of preference) [method @GDScript.assert], [method @GlobalScope.push_error], or [method alert]. See also [method kill].
```
Please use [Oxford comma](https://docs.godotengine.org/en/latest/contributing/documentation/docs_writing_guidelines.html#use-the-oxford-comma-to-enumerate-anything).